static/css/stylesheet.css

```diff
@@ -58,34 +58,35 @@ a{
   padding: 10px;
 }
 
-.navbar .navbar-inner{
-  padding:0
-}
-
 .navbar {
   border: none;
   border-radius: 0;
   margin: 0;
-  display: table;
   width: 100%;
   background-color: #2c3e50;
+  display: flex;
 }
 
-.navbar-nav {
-  margin: 0px;
+.navbar button {
+  border: none;
+  background-color: Transparent;
 }
 
-.navbar-nav .nav ul{
-  width: 100%;
+.navbar button div {
+  width: 20px;
+  height: 2px;
+  background-color: #eeeeee;
+  margin: 5px 0;
 }
 
-.navbar .nav li {
-  width: calc(100vw/5);
-  display: table-cell;
-  float: none;
+.collapse .navbar-collapse{
+  display: flex;
```

Comment:
I kind of like the narrower navbar entries that you changed it to, but I think the dropdown options should be adjusted now:
(unnecessarily wide in comparison)
![image](https://user-images.githubusercontent.com/8433005/29240775-c1012df6-7fa6-11e7-8955-a86eb8ed5891.png)